inventory/hosts

```diff
@@ -198,6 +198,7 @@ uk-prod
 de-prod
 ie-prod
 hu-prod
+ch-prod
```

Comment:
This `europe` list is included in `all-prod` which we use to deploy all managed instances. Since we have a mix of managed and non-managed instances now, we should probably rename these groups. But for now, you need to remove this line if you don't want us to deploy to your instance.

Reply:
done